### ISSUE TITLE
Way to script 3 character press

### ISSUE THREAD

**Issue by rbeede** [hak5darren/USB-Rubber-Ducky#119]
It'd be great to have a way to script 

GUI+CTRL+d

Or any other chained sequence of modifier keys with more than two keys.

The above example creates a new virtual desktop in Windows 10

**Comment by stawrocek** [comment]
Yeah, same problem on Linux with i3 where GUI+SHIFT+Q closes window
